code/controlconfig/controlsconfigcommon.cpp

```diff
@@ -1396,7 +1393,7 @@ void LoadEnumsIntoActionMap() {
 	ADD_ENUM_TO_ACTION_MAP(CUSTOM_CONTROL_5)
 #undef ADD_ENUM_TO_ACTION_MAP
 
-	assert(mActionToVal.size() == CCFG_MAX);
+	Assert(mActionToVal.size() == CCFG_MAX);
```

Comment:
Can we do an assertion here instead?  Helps users get the message that these are errors that need to be reported.

Reply:
Most of the Asserts in the controls code is on hardcode data and not userdata, so if an error occurs its likely due to an oversight done by the coder working on the area.  I've upgraded a number of Assertions in *common to make it more descriptive.